array_test.go

```diff
@@ -1229,6 +1253,18 @@ func TestArray_ContainsOnly(t *testing.T) {
 		value.NotContainsOnly("foo", 123)
 		value.chain.assertFailed(t)
 		value.chain.clearFailed()
+
+		value.ContainsOnly("foo", 123, "bar")
+		value.chain.assertFailed(t)
+		value.chain.clearFailed()
+
+		value.NotContainsOnly("foo", 123, "bar")
+		value.chain.assertNotFailed(t)
+		value.chain.clearFailed()
+
+		value.NotContainsOnly(make(chan int)) // bad type make(chan int)
+		value.chain.assertFailed(t)
+		value.chain.clearFailed()
```

Comment:
Isn't this the same as:

```go
		value.NotContainsOnly(func() {})
		value.chain.assertFailed(t)
		value.chain.clearFailed()
```

which we already have?

Reply:
removed it